servlet/src/main/scala/org/http4s/servlet/BlockingHttp4sServlet.scala

```diff
@@ -0,0 +1,162 @@
+package org.http4s
+package servlet
+
+import java.net.InetSocketAddress
+import javax.servlet._
+import javax.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse, HttpSession}
+
+import cats.effect._
+import cats.implicits.{catsSyntaxEither => _, _}
+import org.http4s.headers.`Transfer-Encoding`
+import org.http4s.server._
+import org.log4s.getLogger
+
+import scala.collection.JavaConverters._
+
+class BlockingHttp4sServlet[F[_]](
+    service: HttpRoutes[F],
+    servletIo: BlockingServletIo[F],
```

Comment:
What's the benefit of parameterizing this?  To configure the chunk size?  I like that it mandates `BlockingServletIo` for this use case.

Reply:
I wanted to keep the `chunkSize` configurable but also mimic the API of `Http4sSevlet` for consistency